docs/reference/query-dsl/nested-query.asciidoc

```diff
@@ -1,12 +1,78 @@
 [[query-dsl-nested-query]]
 === Nested Query
 
-Nested query allows to query nested objects / docs (see
-<<nested,nested mapping>>). The
-query is executed against the nested objects / docs as if they were
-indexed as separate docs (they are, internally) and resulting in the
-root parent doc (or parent nested mapping). Here is a sample mapping we
-will work with:
+A `nested` query allows you to query objects of the <<nested,nested datatype>> type, executing against each field in an array. You can query each field independently while maintaining the relationship between fields in a record, for example, a user's first name, last name, and address.
```

Comment:
For the most part we're breaking around 80 characters

Reply:
Thought I'd set it! Thanks. Fixing.